glotaran/builtin/models/kinetic_spectrum/test/test_spectral_irf.py

```diff
@@ -244,38 +114,27 @@ def test_spectral_irf(suite):
     print(model.validate())
     assert model.valid()
 
-    sim_model = suite.sim_model
-    print(sim_model.validate())
-    assert sim_model.valid()
-
-    wanted_parameters = suite.wanted_parameters
-    print(sim_model.validate(wanted_parameters))
-    print(wanted_parameters)
-    assert sim_model.valid(wanted_parameters)
-
-    initial_parameters = suite.initial_parameters
-    print(model.validate(initial_parameters))
-    assert model.valid(initial_parameters)
-
-    print(model.markdown(wanted_parameters))
+    parameters = suite.parameters
+    print(model.validate(parameters))
+    assert model.valid(parameters)
```

Comment:
I see that you removed the `wanted_parameters` since they were the same as `initial_parameters`, but shouldn't they be different to see if `optimize` works properly?

Reply:
Yes, but this is not an optimizer test, and they were also not different in the initial test (the same parameters were used in both cases). I specifically refactored to make that point clear.